Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -271,7 +276,7 @@ public final class PackageBuilder {
     public static func loadPackage(
         packagePath: AbsolutePath,
         swiftCompiler: AbsolutePath,
-        xcTestMinimumDeploymentTargets: [PackageModel.Platform:PlatformVersion],
```

Comment:
I don't think we want a default value here.

Reply:
This particular method is not used inside SwiftPM. It is aimed at clients who want to load the package to see what it contains. If this parameter _shouldn’t_ have a default value, we should explain in the example client what it is and why it’s needed. At the moment it doesn’t even have an entry in the method documentation. What should the example client be passing [here](https://github.com/apple/swift-package-manager/blob/07e42c3b4ee2237ab574fcbb4c73d14c5dacc5c2/Examples/package-info/Sources/package-info/main.swift#L35)?

@dduan noticed this too over in #2757. Both of us assumed a client should just pass `[:]` for most purposes, but maybe we misunderstand something?

Reply:
Ah, I think I mixed up this method and the `PackageBuilder` initializer. I think we should be using `MinimumDeploymentTarget.default.xcTestMinimumDeploymentTargets` as the default, so that we are consistent with the default that `PackageGraphLoader` is using. When the method is used by unit tests, we should be passing an empty dictionary, though.

Reply:
Ah, nice. That makes sense. Done.